src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1202,7 +1213,9 @@ format layout_optimizer::get_expected_format(convolution_node const& node) {
                   || (_optimized_conv_count.at({format::fs_b_yx_fsv32, false}) * cond_denom > 0.8f)) &&
                   convolution_fs_b_yx_fsv32_opt(input_layout,
                                                 output_layout,
-                                                weights_layout, prim, true))))) {
+                                                weights_layout, prim, true)))) &&
+                 !(has_rank3_mvn_user(reinterpret_cast<program_node const&>(*node.get_users().front()), 0, 3, 1000000) &&
+                     !static_cast<bool>(prepare_padding::get_convolution_needed_padding(const_cast<convolution_node&>(node))))) {
```

Comment:
`node` is already of type `convolution_node`. Do we need the `const_cast`?

Reply:
Need const_cast. node is `const convolution_node` in here, and `get_convolution_needed_padding` needs non const type. `get_output_layout()` used in `get_convolution_needed_padding`, has differently implemented const and non const API.  CI failure in unit-test with const case.